release_notes/Service_Fabric_ReleaseNotes_90.md

```diff
@@ -0,0 +1,153 @@
+| 14. | **Feature** | Create replacement replicas before Replica Restart Wait Duration (RRWD) expires when there are less than min replicas up | **Brief Description**: Starting with Service Fabric version 9.0, we have ability to adjust logic around Replica Restart Wait Duration.<br>Currently, when replica goes down, SF waits for a configurable time period until decides to build a new replica elsewhere, hoping the existing replica might become available again.<br>The problem here is that, when partition goes below MinReplicaSetSize number of replicas, it will still wait for that period of time before bringing new replica up, causing unavailability for the customer.<br>By toggling IgnoreReplicaRestartWaitDurationWhenBelowMinReplicaSetSize on, SF will not wait for problematic replica, but start building new one instantly, making sure that there is at least MinReplicaSetSize replicas available as soon as possible.<br>This configuration is dynamic, and customer can toggle the behavior accordingly.<br>The default value for this config is false, meaning that it is switched off.<br>**Impact**: SQL MI asked for the feature as a way to increase availability.<br> |
+| 15. | **Feature** | Enforce a configurable Max value on the InstanceCloseDelayDuration | **Brief Description**: Enforcing a configurable Max value on the InstanceCloseDelayDuration to prevent users from inadvertently specifying huge, delay values which could result in updates getting stuck.<br>Checks have been added in service creation/updation, app upgrade, application & fabric upgrade overrides to prevent users from configuring anything more than the max config value.<br>The max value is a cluster configuration, defaulting to 30mins, so an InstanceCloseDelay can't be specified more than this value. Users can modify this value to anything as they require in the cluster config.<br>For already misconfigured services, the delay would be ceiled to the max config value.<br>**Documentation Reference**: For more about it at <ul><li>[Service Fabric application upgrade: Advanced topics](https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-application-upgrade-advanced)</li></ul>**Breaking Change**: This would be a **breaking change** if customers are already configuring more than 30 mins delay.<br>**Workaround**: They would need to define a higher value in the cluster config. |
```

Comment:
```suggestion
| 15. | **Feature** | Enforce a configurable Max value on the InstanceCloseDelayDuration | **Brief Description**: Enforcing a configurable Max value on the InstanceCloseDelayDuration to prevent users from inadvertently specifying huge delay values which could result in updates getting stuck.<br>Checks have been added in service creation/updation, app upgrade, application and fabric upgrade overrides to prevent users from configuring anything more than the max config value.<br>The max value is a cluster configuration, defaulting to 30mins, so an InstanceCloseDelay can't be specified more than this value. Users can modify this value to anything they require in the cluster config.<br>For already misconfigured services, the delay would be ceiled to the max config value.<br>**Documentation Reference**: For more about it at <ul><li>[Service Fabric application upgrade: Advanced topics](https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-application-upgrade-advanced)</li></ul>**Breaking Change**: This would be a **breaking change** if customers are already configuring more than 30 mins delay.<br>**Workaround**: They would need to define a higher value in the cluster config. |
```

Reply:
^rephrasing: "&"-->"and", "anything as they require"-->"anything they require", removed unnecessary comma after "huge"